### DISCUSSION TITLE
may I ask how skywalking client js can be connected to React

### DISCUSSION THREAD

**Discussion by 13992764153** [apache/skywalking#12473] [Q&A]
Hello, may I ask if there is any documentation related to skywalking client js supporting the React framework? How can I access it in the React framework

**Comment by wu-sheng** [comment]
https://github.com/apache/skywalking-client-js/tree/v0.12.0?tab=readme-ov-file#catching-errors-in-frames-including-react-angular-vue

**Reply by 13992764153** [reply]
thank you，Not reading the readme in a timely manner
